patches/minecraft/net/minecraft/entity/EntityLivingBase.java.patch

```diff
@@ -141,7 +173,33 @@
              }
  
              this.field_70170_p.func_72960_a(this, (byte)3);
-@@ -1253,15 +1273,7 @@
+@@ -1195,19 +1222,22 @@
+ 
+     public void func_70653_a(Entity p_70653_1_, float p_70653_2_, double p_70653_3_, double p_70653_5_)
+     {
++        LivingKnockBackEvent event = net.minecraftforge.common.ForgeHooks.onLivingKnockBack(this, p_70653_1_, p_70653_2_, p_70653_3_, p_70653_5_);
```

Comment:
You should use a fully qualified name, rather than an import here; `net.minecraftforge.event.entity.living.LivingKnockBackEvent event = ...`